zebra-instructions.md

```diff
@@ -74,18 +77,72 @@ These fixes disable mining pool operator payments and miner payments: they just
     [mining]
     miner_address = 't27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v'
     ```
+
     </details>
+
 3. [Run Zebra](https://zebra.zfnd.org/user/run.html) with the `getblocktemplate-rpcs` feature:
+
     ```sh
     cargo run --release --features "getblocktemplate-rpcs"
     ```
+
 4. Wait a few hours for Zebra to sync to the testnet tip (on mainnet this takes 2-3 days)
 
-## Install and run a mining pool
+## Install `s-nomp`
+
+### Arch-specific instructions
+
+#### Install dependencies:
+
+1. Install [`redis`](https://redis.io/docs/getting-started/) and run it on the default port:
+
+    ```sh
+    sudo pacman -S redis
+    sudo systemctl start redis
+    ```
+
+2. Install and activate [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating):
+
+    ```sh
+    sudo pacman -S nvm
+    unset npm_config_prefix
+    source /usr/share/nvm/init-nvm.sh
+    ```
+
+3. Install `boost` and `libsodium` development libraries:
+
+    ```sh
+    sudo pacman -S boost libsodium
+    ```
+
+#### Install `s-nomp`:
+
+1. `git clone https://github.com/ZcashFoundation/s-nomp && cd s-nomp`
+
+2. Use the Zebra configs: `git checkout zebra-mining`
+
+    - TODO: change the fork in `package.json` to ZcashFoundation
+
+3. Use node 8.11.0:
+
+    ```sh
+    nvm install 8.11.0
+    nvm use 8.11.0
+    ```
+
+4. Update dependencies and install:
+
+    ```sh
+    npm update
+    npm install
+    ```
+
+### Ubuntu-specific and general instructions
```

Comment:
These instructions are not Ubuntu-specific, they should work for any Debian-derived distro using `apt`. (This was an issue in the existing doc.)

So I'd like us to say Debian/Ubuntu.

```suggestion
### General instructions with Debian/Ubuntu examples
```